src-tauri/src/app.rs

```diff
@@ -86,18 +87,19 @@ impl Service {
     ///
     /// The node and several channel senders are added to the shared app context while channel
     /// receivers are stored on the Service struct for use during the runtime loop.
-    pub async fn build(blobs_base_dir: PathBuf) -> anyhow::Result<Self> {
-        let private_key = PrivateKey::new();
+    pub async fn build(app_data_dir: PathBuf) -> anyhow::Result<Self> {
+        // Use an ephemeral private key if we're running with `tauri dev`.
+        let private_key = if cfg!(dev) {
+            PrivateKey::new()
+        } else {
+            let private_key_path = app_data_dir.join("private_key.txt");
```

Comment:
`private_key.txt` seemed like an obvious choice here but I'm open to suggested changes.